qemu_v8.mk

```diff
@@ -498,6 +514,36 @@ else
 QEMU_MTE	= off
 endif
 
+QEMU_ARGS = -nographic
+QEMU_ARGS += -smp $(QEMU_SMP)
+QEMU_ARGS += -cpu $(QEMU_CPU)
+QEMU_ARGS += -d unimp -semihosting-config enable=on,target=native
+QEMU_ARGS += -m $(QEMU_MEM)
+QEMU_ARGS += -bios bl1.bin
+QEMU_ARGS += -initrd rootfs.cpio.gz
+QEMU_ARGS += -kernel Image
+QEMU_ARGS += -append 'console=ttyAMA0,38400 keep_bootcon root=/dev/vda2 $(QEMU_KERNEL_BOOTARGS)'
+QEMU_ARGS += $(QEMU_XEN)
+QEMU_ARGS += $(QEMU_EXTRA_ARGS)
+QEMU_ARGS += -s -S -machine virt,acpi=off,secure=on,mte=$(QEMU_MTE),gic-version=$(QEMU_GIC_VERSION),virtualization=$(QEMU_VIRT)
+
+ifeq ($(WITH_SCMI),y)
+QEMU_DUMP_ARGS = $(QEMU_ARGS),dumpdtb=qemu_v8.dtb
+
+QEMU_SCMI_ARGS 	= -dtb $(SCMI_DTB)
+
+$(SCMI_DTBO): $(SCMI_DTSO)
+	dtc -I dts -O dtb -o $(SCMI_DTBO) $(SCMI_DTSO)
+
+$(SCMI_DTB): $(SCMI_DTBO) .stamp_qemu
+	cd $(BINARIES_PATH) && $(QEMU_BUILD)/aarch64-softmmu/qemu-system-aarch64 \
+		$(QEMU_DUMP_ARGS)
+	cd $(BINARIES_PATH) && fdtoverlay -i qemu_v8.dtb -o $(SCMI_DTB) $(SCMI_DTBO)
+endif
+
+QEMU_RUN_ARGS = $(QEMU_ARGS) $(QEMU_SCMI_ARGS)
```

Comment:
How does that propagate into the QEMU arguments when running `make check`? I think the `check:` target should be refactored too in order to use `QEMU_RUN_ARGS` (the `-serial` arguments would be left out and kept in the `run-only` target and in `qemu-check.exp` since they differ).

Reply:
Ok, I didn't noticed nor used the 'make check'. What is the purpose ?

I'm going to have a look

Reply:
`make check` runs `xtest` and other runtime tests for OP-TEE. Unlike `make run`, it does not require several terminals and does not print out a lot of information during the tests; instead it prints hash sign (`#`) for each successful `xtest` sub-case. It is useful during development but more importantly for CI where a final status (PASS/FAIL) is required.
```
$ make -j$(nproc) check
[... lots of build logs...]
Starting QEMU... done, guest is booted.
Running: xtest ...
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
##################################################
###################################
Status: PASS (1435 test cases)
Running: keyctl tests...
Status: keyctl tests successful
Test Rust example applications:
Running acipher-rs...
Test success
Running aes-rs...
Test success
Running authentication-rs...
Test success
Running big_int-rs...
Test success
Running diffie_hellman-rs...
Test success
Running digest-rs...
Test success
Running hello_world-rs...
Test success
Running hotp-rs...
Test success
Running random-rs...
Test success
Running secure_storage-rs...
Test success
Running supp_plugin-rs...
Test success
Running time-rs...
Test success
Running signature_verification-rs...
Test success
Rust examples tested successfully
$ echo $?
0
```